modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -54,8 +54,14 @@ public TypeScriptFetchClientCodegen() {
         outputFolder = "generated-code/typescript-fetch";
         embeddedTemplateDir = templateDir = "typescript-fetch";
 
-        this.apiPackage = "src" + File.separator +"apis";
-        this.modelPackage = "src" + File.separator + "models";
+        if (additionalProperties.containsKey(NPM_NAME)) {
```

Comment:
It seems like I cannot access additionalProperties in the class constructor. Any suggestions about a better way to do this?

Reply:
Resolved: This can be handled in processOpts()